pages/meet/index.tsx

```diff
@@ -1,4 +1,5 @@
 import React, { ReactElement } from 'react';
+import Quote from '../../components/quote/Quote';
```

Comment:
Would you create an index.js inside src/components/quote and import the quote component there and then export it from there as well.  You can look inside the other component folders to see what I mean.  This way we can clean up the import here to 
`import Quote from '../../components/quote`